src/sortStudents.ts

```diff
@@ -1,16 +1,71 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
+
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades'
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+export function sortStudents(
+  students:Student[],
+  sortBy:SortType,
+  order:SortOrder,
```

Comment:
```suggestion
  students: Student[],
  sortBy: SortType,
  order: SortOrder,
```
better to separate by space))